lib/dash/dash_parser.js

```diff
@@ -1059,6 +1059,11 @@ shaka.dash.DashParser = class {
       context.representation.mimeType = 'audio/eac3-joc';
     }
 
+    let forced = false;
+    if (isText) {
+      forced = roles.includes('forced_subtitle');
```

Comment:
I would suggest a comment to the DASH-IF bug that adds this.  This doesn't appear in any spec yet, so the bug is the best we can do.